pyOCD/tools/flash_tool.py

```diff
@@ -77,7 +86,8 @@
 group.add_argument("-se", "--sector_erase", action="store_true", help="Use sector erase when programming.")
```

Comment:
Yes, I like separating the action from the programming algorithm. But I'll leave that for another time. :)